### PR TITLE
Changes in 8.17+rc1.

### PR DESCRIPTION
**Pull request by Zimmi48**

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -8,6 +8,539 @@ Recent changes
...
+  the `skipn_skipn` lemma in `Lists/List.v`
+  (`#16632 <https://github.com/coq/coq/pull/16632>`_,
+  by Stephan Boyer).
+- **Added:**
+  lemmas :g:`nth_error_ext`, :g:`map_repeat`, :g:`rev_repeat` to ``List.v``,
+  and :g:`to_list_nil_iff`, :g:`to_list_inj` to ``VectorSpec.v``
+  (`#16756 <https://github.com/coq/coq/pull/16756>`_,
+  by Stefan Haan).
+- **Added:** transparent :g:`extgcd` to replace opaque :g:`euclid`,
+  :g:`euclid_rec', :g:`Euclid`, and :g:`Euclid_intro` in :g:`Znumtheory`.
```
Comment (olaure01) [inline doc/sphinx/changes.rst:462]:
```suggestion
  :g:`euclid_rec`, :g:`Euclid`, and :g:`Euclid_intro` in :g:`Znumtheory`.
```

Comment (Zimmi48) [reply doc/sphinx/changes.rst:462]: Thanks!

File: doc/sphinx/language/extensions/match.rst [hunk 2]
```diff
@@ -316,21 +317,13 @@ a warning is issued in such situations.
...
-      .. coqtop:: none
-
-         Set Warnings "-unused-pattern-matching-variable".
-
-      .. coqtop:: all
+      .. coqtop:: all warn
 
          Definition is_zero (o : option nat) := match o with
-         | Some 0 => true
+         | Some _ => true
```
Comment (Alizter) [inline doc/sphinx/language/extensions/match.rst:323]: This is a dumb example. Something like:
```coq
Definition is_some (o : option nat) :=
  match o with
  | Some x => true
  | None => false
  end.
```
is better

Comment (Zimmi48) [reply doc/sphinx/language/extensions/match.rst:323]: This one doesn't trigger the warning though.

Comment (Alizter) [reply doc/sphinx/language/extensions/match.rst:323]: Ah ok then I misread.

**Comment by Zimmi48** [comment]
@jfehrle Would you mind assigning / merging this PR?

**Review by jfehrle — APPROVED** [review]
Rubber stamped.

@coqbot: merge now

**Comment by jfehrle** [comment]
@coqbot: merge now

**Pull request merged**